docs/src/providers/querying.md

```diff
@@ -0,0 +1,41 @@
+# Interacting with the blockchain
+
+Once you have set up a provider, you're ready to interact with the Fuel blockchain. Here are a few examples of what you can do with a provider; for a more in-depth overview of the API, check the [official provider API documentation](https://docs.rs/fuels/latest/fuels/signers/provider/struct.Provider.html).
+
+- [Interacting with the blockchain](#interacting-with-the-blockchain)
+  - [Set up](#set-up)
+  - [Get all coins from an address](#get-all-coins-from-an-address)
+  - [Get spendable coins from an address](#get-spendable-coins-from-an-address)
+  - [Get balances from an address](#get-balances-from-an-address)
+
+## Set up
+
+You might need to set up a test blockchain first. You can skip this step if you're connecting to an external blockchain.
+
+```rust,ignore
+{{#include ../../../examples/providers/src/lib.rs:setup_test_blockchain}}
+```
+
+## Get all coins from an address
+
+This method returns all coins from a wallet, including spent ones.
+
+```rust,ignore
+{{#include ../../../examples/providers/src/lib.rs:get_coins}}
+```
+
```

Comment:
I don't know, I'd rather _not_ assume what the user will or will not do with the SDK. I believe we should make all public APIs in the SDK easily visible/searchable. What if someone is trying to build a chain scrapper? They would want to know about this. And since this is in fact querying the chain, it should be in `querying.md` IMO.